zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -1032,15 +1032,22 @@ def get_mon_count(self, unit):
         match = rx.search(result['Stdout'])
         return int(match.group(1))
 
+    def get_mon_units(self):
+        """Get a list of the ceph-mon units."""
+        return list(x.entity_id for x in zaza_model.get_units('ceph-mon'))
```

Comment:
It would be slightly more efficient to use a list comprehension rather than a generator and then consume that to a list, but I realise it was already like this previously.